scripts/e2e-test.sh

```diff
@@ -39,16 +94,43 @@ function boskosctl_wrapper() {
 
 # Acquire GCP project from Boskos, the manager of projects pool
 # Return the project name being acquired
+# $1 ... boskos project type
 function boskos_acquire() {
     local resource
     local resource_name
     echo "Try to acquire project from boskos" >&2
-    resource="$(boskosctl_wrapper acquire --type verification-smoke-project --state free --target-state busy --timeout 10m)"
+    resource="$(boskosctl_wrapper acquire --type $1 --state free --target-state busy --timeout 10m)"
     resource_name="$(jq .name <<<"${resource}" | tr -d \")"
     echo "Acquired project from boskos: ${resource_name}" >&2
     # Send a heartbeat in the background to keep the lease while using the resource.
     boskosctl_wrapper heartbeat --resource "${resource}" >/dev/null 2>&1 &
     echo "${resource_name}"
 }
 
-main
+# help prints help.
+function help() {
+  echo 1>&2 "Usage: ${PROGNAME} <command>"
+  echo 1>&2 ""
+  echo 1>&2 "Commands:"
+  echo 1>&2 "  smoke        terraform smoke test"
+  echo 1>&2 "  incremental  incremental e2e test"
+}
+
+#ACTION="${1:-}"
```

Comment:
Intended to be commented?

Reply:
Yes. Need to merge https://github.com/GoogleCloudPlatform/oss-test-infra/pull/546 first